### ISSUE TITLE
Wifi pop-up in the waybar minor issue ; nm-applet

### ISSUE THREAD

**Issue by UcraNerd** [prasanthrangan/hyprdots#236]
![230918_18h23m51s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/102480084/248e4ad4-a6d0-4be0-a882-a8bb731994fe)

Some buttons are deformed... idk why. <" ">

**Comment by zer0cha0s** [comment]
Someone please correct me if I'm wrong but this is the tray section of Waybar so that context menu has nothing to do with this setup and is probably an issue with the application itself. 
The bluetooth icon i have next to network has a perfectly formed and aligned context menu icons.

**Comment by UcraNerd** [comment]
> Someone please correct me if I'm wrong but this is the tray section of Waybar so that context menu has nothing to do with this setup and is probably an issue with the application itself. 
> The bluetooth icon i have next to network has a perfectly formed and aligned context menu icons.

Maybe you're right, still I don'tknow how to solve it...

**Comment by prasanthrangan** [comment]
yup, i'm aware of this in networkmanager, is there any other app that has such buttons?

**Comment by katsuja** [comment]
> yup, i'm aware of this in networkmanager, is there any other app that has such buttons?



yup, i'm aware of this in networkmanager, is there any other app that has such buttons?
![230920_00h43m42s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/ee02c360-c34a-45c3-9179-7532ea94ca0b)

**Comment by T-Crypt** [comment]
What is your scaling? I had issues with the pop-ups previously based on the scaling of my resolution set in the monitors.conf..

Can you adjust the scaling and confirm that does or doesn't resolve this?

**Comment by UcraNerd** [comment]
> What is your scaling? I had issues with the pop-ups previously based on the scaling of my resolution set in the monitors.conf..
> 
> Can you adjust the scaling and confirm that does or doesn't resolve this?

My monitors.conf file is blank... and i don't have a second monitor so idk

**Comment by T-Crypt** [comment]
Can you use hyprctl monitors and paste your output here? should show the scaling you have set.

**Comment by UcraNerd** [comment]
I have hyprland.conf with monitor=eDP-1, 1920x1080@60, 0x0, 1

**Comment by katsuja** [comment]
> What is your scaling? I had issues with the pop-ups previously based on the scaling of my resolution set in the monitors.conf..
> 
> Can you adjust the scaling and confirm that does or doesn't resolve this?

He had no problems before.
I reveal this problem on both laptops (intel igpu) and desktops (amd dedicated gpu).
I attach screenshots of 4 tests done with different scaling:
0.9 - 1 - 1.1 - 1.2
![230922_23h35m19s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/f7039b3a-205f-457c-9d2a-4fd02a95eae9)
![230922_23h35m37s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/e8c57b9a-c22b-49bd-93dc-297ef4ea7e37)
![230922_23h35m56s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/ef1c52fe-3c31-4917-82a0-11c0a089678a)
![230922_23h36m13s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/544af197-6cae-48dd-aecf-d8960c87769b)

**Comment by T-Crypt** [comment]
yeah its either an issue with the applet itself I have found or with the style.css for the waybar but not convinced as it is not apparent on all the applets available.. 

The nm-applet is not that reliable from what I have read.

**Comment by Muhammad-Abubakr** [comment]
> > yup, i'm aware of this in networkmanager, is there any other app that has such buttons?
> 
> Same problem with the redshift-gtk icon
> 
> ![230920_00h46m23s_screenshot](https://user-images.githubusercontent.com/20012885/269112716-a5d9edee-161a-4d05-b4ed-e247b570a88d.png)

I am sorry if this is irrelevant but how did you manage to get redshift working on wayland?

**Comment by katsuja** [comment]
redshift with geoclue2
![231028_14h58m48s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/20012885/38fe1e7f-4c05-4e89-a596-7dbeb6a762ac)
take a look here:
https://wiki.archlinux.org/title/redshift

**Comment by Muhammad-Abubakr** [comment]
> redshift with geoclue2 ![231028_14h58m48s_screenshot](https://user-images.githubusercontent.com/20012885/278814739-38fe1e7f-4c05-4e89-a596-7dbeb6a762ac.png) ![231028_15h01m52s_screenshot](https://user-images.githubusercontent.com/20012885/278814870-7741262e-58bf-4ca1-918e-d3b443402b6e.png)
> 
> take a look here: https://wiki.archlinux.org/title/redshift If you have a laptop with an ambient light sensor then use Clight: https://github.com/FedeDP/Clight

Thank you for the reply, but as Arch Wiki states redshift has no support for wayland right now. I had installed and configured it with geoclue2 but the temprature didn't seem to change

**Comment by katsuja** [comment]
> ambient light sensor

It's actually running and systemctl doesn't report any errors but the temperature doesn't actually change. I did not notice that.
I had created an arch_foreign_pkgs.txt file from an old i3 installation. I uninstalled nothing!
  Now I have to recheck some core from the aforementioned file.
However cligth works great with laptops with an ambient light sensor. It also manages temperatures very well.
Sorry for the wrong information 😁
